[formrecognizer] Update code to language code

catalinaperalta opened the pull request:
Update code to language_code on DocumentLanguage

azure-sdk commented:
API changes have been detected in `azure-ai-formrecognizer`. You can review API changes [here](https://apiview.dev/Assemblies/Review/d71b75a65e3a481ba57cd0ce5a704cbf?diffOnly=True&diffRevisionId=5a3f38fe1dc94efea303b698d3cf890a)

**API changes**
```diff
-     ivar code: str
+     ivar language_code: str
```

kristapratico reviewed (approved):
LGTM. The prebuilt-read samples need rename too like here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/sample_analyze_read.py#L58

catalinaperalta commented:
> LGTM. The prebuilt-read samples need rename too like here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/sample_analyze_read.py#L58

Thanks for the reminder! Fixed it. Also fixed the path issue.
